src/disasm.cpp

```diff
@@ -1172,7 +1178,11 @@ class LineNumberPrinterHandler : public AsmPrinterHandler {
         Buffer.clear();
     }
 
+#if JL_LLVM_VERSION >= 190000
+    virtual void setSymbolSize(const MCSymbol *Sym, uint64_t Size) {}
+#else
     virtual void setSymbolSize(const MCSymbol *Sym, uint64_t Size) override {}
+#endif
     //virtual void beginModule(Module *M) override {}
```

Comment:
Dropping override is typically very concerning and indicates this part of the change is wrong. Notably, AsmPrinterHandler was renamed to DebugHandlerBase and addAsmPrinterHandler was renamed addDebugHandler so that a new incompatible API could use the existing names. (117b53ae38428ca66eaa886fb432e6f09db88fe4)